docs/en/OLED_Display.md

```diff
@@ -0,0 +1,182 @@
+# Oled Display
+More user friendly extension than PEG Oled for using OLED display in your build.
+
+If you need help soldering, there's a small note on the end to help.
+
+# Preparation
+
+### Libraries
+First of all you need to download few Libraries that will make possible for your OLED to work. You can download them from Adafruit CircuitPython Libraries page. 
+#### Make sure you download right version of Libraries. It should be the same one as yours CircuitPyhon.
+* [CircuitPython Libraries Bundle](https://circuitpython.org/libraries)
+
+Now you need to create `lib` folder where you will place your needed files.
+Then find following folder and file and drop them in your freshly baked folder.
+* adafruit_display_text
+* `adafruit_displayio_ssd1306.mpy`
+
+# Main.py
+Time to make changes in your `main.py`.
+As always, first step is adding OLED extension.
+
+```python
+from kmk.extensions.oled import Oled, TextEntry, ImageEntry
+```
+
+Now add this main part of extension. Then replace SCL and SDA. You can also tweak some settings of your screen here, but for now its not necessary.
+
+```python
+i2c_bus = busio.I2C(board.GP SCL, board.GP SDA) # change SCL and SDA according to your board and made connection.
+oled = Oled(
+    i2c=i2c_bus,
+    device_address=0x3C,
+    height=64, # here you can change your screen size
+    flip=False,
+    dim_time=10,
+    dim_target=0.1, 
+    off_time=0, 
+    brightness=0.1,
+    brightness_step=0.1,
```

Comment:
There's also width and I2C device address. Flip can be set on per-side basis if the keyboard is split, there are powersave versions of arguments that are only active when powersave is enabled. Each argument should be described.

Brightness step relates brightness increase/decrease keycodes, which are currently broken, but it should be an easy fix.